source/main.cc

```diff
@@ -366,6 +366,11 @@ read_parameter_file(const std::string &parameter_file_name)
       std::ifstream parameter_file(parameter_file_name.c_str());
       if (!parameter_file)
         {
+          if (parameter_file_name=="parameter-file.prm")
+            {
+              std::cout << "*** https://www.youtube.com/watch?v=dQw4w9WgXcQ ***" << std::endl;
```

Comment:
Will this become the new commit jingle? Or did you hope to hide this between all the obscure lambda syntax? :smile:
Anyway, I would prefer either a more serious error message, or we just remove this code. How likely is it that somebody stumbles into this? And even if so, how likely is it that we break somebody else's models?